lib/taskana-core/src/test/java/acceptance/task/TransferTaskAccTest.java

```diff
@@ -0,0 +1,92 @@
+package acceptance.task;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.sql.SQLException;
+
+import org.h2.store.fs.FileUtils;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import acceptance.AbstractAccTest;
+import pro.taskana.Task;
+import pro.taskana.TaskService;
+import pro.taskana.exceptions.ClassificationNotFoundException;
+import pro.taskana.exceptions.InvalidArgumentException;
+import pro.taskana.exceptions.InvalidWorkbasketException;
+import pro.taskana.exceptions.NotAuthorizedException;
+import pro.taskana.exceptions.TaskAlreadyExistException;
+import pro.taskana.exceptions.TaskNotFoundException;
+import pro.taskana.exceptions.WorkbasketNotFoundException;
+import pro.taskana.security.JAASRunner;
+import pro.taskana.security.WithAccessId;
+
+/**
+ * Acceptance test for all "update task" scenarios.
+ */
+@RunWith(JAASRunner.class)
+public class TransferTaskAccTest extends AbstractAccTest {
+
+    public TransferTaskAccTest() {
+        super();
+    }
+
+    @WithAccessId(
+            userName = "user_1_1",
+            groupNames = { "group_1" })
+    @Test
+    public void testDomainChangingWhenTransferTask()
+            throws SQLException, NotAuthorizedException, InvalidArgumentException, ClassificationNotFoundException,
+            WorkbasketNotFoundException, TaskAlreadyExistException, InvalidWorkbasketException, TaskNotFoundException {
+
+        TaskService taskService = taskanaEngine.getTaskService();
+
```

Comment:
Please write codes in blocks in this calss.
Code is simple and can be written as 1-2 blocks (by functionallity seperated) - Have a look the other lines in this class.